src/avatar-group.scss

```diff
@@ -0,0 +1,103 @@
+@import "./mixins";
+@import "./avatar";
+
+/*!
+.fd-avatar-group+((--individual|--group), (--xxs|--xs|--s|--m|--l|--xl|--xxl)
+*/
+$block: #{$fd-namespace}-avatar-group;
+
+.#{$block} {
+  $fd-avatar-group-item-spacing: (
+    "individual": (
+      "xs": ("margin-right": 0.0625rem),
+      "s": ("margin-right": 0.125rem),
+      "m": ("margin-right": 0.125rem),
+      "l": ("margin-right": 0.125rem),
+      "xl": ("margin-right": 0.25rem)
+    ),
+    "group": (
+      "xs": ("margin-right": -0.75rem),
+      "s": ("margin-right": -1.25rem),
+      "m": ("margin-right": -1.625rem),
+      "l": ("margin-right": -2rem),
+      "xl": ("margin-right": -2.75rem)
+    )
+  );
+
+  $fd-const-ratio: 1rem;
```

Comment:
nit: Maybe it's better not to separate variables with an empty line?
For me, it's kind of a logical separation and it's a bit confusing.